docs/model_developers/configurationmodel.rst

```diff
@@ -12,73 +13,62 @@ contains the source code of the configuration model. This project also matches w
 :term:`project` defined on the server, from which multiple :term:`environments<environment>` can be
 deployed.
 
-.. note::
-
-    Inmanta requires that a project is a git repository. This is not strictly required when using
```

Comment:
Is this information still present somewhere in the documentation?

Reply:
added part of the note back